### PR TITLE
added the two new classes for AT100

### PR DESCRIPTION
**Pull request by voisardf**
added new classes for AT100

### CODE REVIEW

**Comment by voisardf** [comment]
@maltaesousa : just two new classes and their symbol. Tested locally

File: mapserver/crdppf.map.mako [hunk 1]
```diff
@@ -1056,6 +1056,26 @@ LAYER
         COLOR  157 159 162
     END #STYLE
   END #CLASS
+  CLASS
+    NAME "Zone d'utilité publique 1 - Constructible"
+    EXPRESSION /1511/
```
Comment (maltaesousa) [inline mapserver/crdppf.map.mako:1061]:
```suggestion
    EXPRESSION 1511
```
Do not use regex when it isn't needed. They perform badly :)

File: mapserver/crdppf.map.mako [hunk 2]
```diff
@@ -1056,6 +1056,26 @@ LAYER
...
+    STYLE
+        COLOR  157 159 162
+    END #STYLE
+  END #CLASS
+  CLASS
+    NAME "Zone d'utilité publique 2 - À constructibilité restreinte"
+    EXPRESSION /1611/
```
Comment (maltaesousa) [inline mapserver/crdppf.map.mako:1068]:
```suggestion
    EXPRESSION 1611
```

File: mapserver/crdppf.map.mako [hunk 3]
```diff
@@ -1056,6 +1056,26 @@ LAYER
...
+    STYLE
+        SYMBOL "hatchsymbol"
+        SIZE 6
+        COLOR 157 159 162
+        WIDTH 3
+    END #STYLE
+  END #CLASS
   CLASS
     NAME "Zone de verdure"
     EXPRESSION /1603|1613/
```
Comment (maltaesousa) [inline mapserver/crdppf.map.mako:1081]:
```suggestion
    EXPRESSION {1603,1613}
```

Check here : https://mapserver.org/mapfile/expressions.html#list-expressions

**Review by maltaesousa — APPROVED** [review]
Not very important but just a notice on regular expressions

File: mapserver/crdppf.map.mako (hunk 3 above)
Comment (voisardf) [reply mapserver/crdppf.map.mako:1081]: thanks! I wanted to do that anyway! 👍 
Still, I merge this PR to apply the changes to production. I'll push another PR later in the afternoon with the cleaned mapfile.

**Pull request merged**